spec/features/login_signup/login_spec.rb

```diff
@@ -18,31 +18,5 @@
     expect(page).to have_link('Sign Up')
   end
 
-  context 'Form Submission' do
-    scenario 'Submit form without email and password' do
-      click_button 'Log In'
-      expect(page).to have_content 'Invalid Email or password.'
-    end
-
-    scenario 'Submit form with incorrect email and password' do
-      within 'form' do
-        fill_in 'Email', with: 'newmail@gmail.com'
-        fill_in 'Password', with: '123456'
-      end
-      click_button 'Log In'
-      expect(page).to have_content 'Invalid Email or password.'
-    end
-
-    scenario 'Submit form with correct email and password' do
-      @user = User.create(name: 'Dammy', email: 'admin@gmail.com', password: 'password')
-      within 'form' do
-        fill_in 'Email', with: @user.email
-        fill_in 'Password', with: @user.password
-      end
-      click_button 'Log In'
-      expect(page).to have_current_path(authenticated_root_path)
-      expect(page).to have_content 'Signed in successfully.'
-    end
-  end
```

Comment:
- [ ] It's great that you have implemented some of the requirements suggested by the previous reviewer :+1: but I guess he told you to fix these tests instead of removing them :fearful: Kindly make them pass because tests are so much important for your application because they told you where the error is if your application suddenly breaks or something unexpected happens :thinking: